public/js/app.js

```diff
@@ -32,8 +34,20 @@ store.dispatch({
     receivedAt: Date.now()
 });
 
-render(
-    <Provider store={store}>
-      {routes}
-    </Provider>
-    , document.getElementById('react-mount'));
+const render = (Component) => {
+  ReactDOM.render(
+    <AppContainer>
+      {Component}
+    </AppContainer>,
+    document.getElementById('react-mount')
+  );
+};
+
+
+render(<BaseApp store={store} />);
+
+if (module.hot) {
```

Comment:
This is now required... we're listening for the hot module changes and triggering a re-render